tests/boot/with_mcumgr/overlay-mcuboot-version.conf

```diff
@@ -0,0 +1,2 @@
+# Set version of MCUBOOT bypassing extra options to imgtool
+CONFIG_MCUBOOT_EXTRA_IMGTOOL_ARGS="--version 1.1.1+1"
```

Comment:
Can do this with a `VERSION` file instead

Reply:
I pass this by the `overlay-config`, because it is used only in one subtest in `testcase.yaml:boot.with_mcumgr.test_downgrade_prevention`:
https://github.com/zephyrproject-rtos/zephyr/blob/95e1df0d430ac90e3fa148dd67860a80df0bffa1/tests/boot/with_mcumgr/testcase.yaml#L30
I tried to pass it directly in `testcase.yaml` without overlay file, using `EXTRA_ARGS` ... but there was a problem with double quotes (they are removed by twister logic and not passed to cmake args).

Reply:
I found there is also `MCUBOOT_IMGTOOL_SIGN_VERSION` config, but still I cannot use it in `testspec.yaml` as `extra_args`, because Twister removes double quotes (maybe issue to Twister to fix it),
Also I tried with `extra_config` option in Twister yaml, but looks, that this option does not support sysbuild yet (an overlay file is created and `SB_` and `mcuboot_` prefixes are not removed)

Reply:
```
    extra_args:
      - mcuboot_CONFIG_MCUBOOT_DOWNGRADE_PREVENTION=y
    extra_configs:
      - CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION="1.1.1+1"
```
but combination of extra_configs and extra_args works good. I removed this overlay file